gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/DuplicateObservationStatementMapper.java

```diff
@@ -45,7 +45,20 @@ private static void mergeDuplicateObservationStatements(RCMRMT030101UKEhrComposi
             .map(Optional::get)
             .filter(DuplicateObservationStatementMapper::hasSinglePertinentInformation)
             .filter(DuplicateObservationStatementMapper::isAnnotationTruncated)
-            .forEach(truncatedObservationStatement ->
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getPriorityCode() == null)
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getUncertaintyCode() == null)
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getValue() == null)
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getInterpretationCode() == null)
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getSubject() == null)
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getSpecimen() == null || obs.getSpecimen().isEmpty())
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getReferenceRange() == null || obs.getReferenceRange().isEmpty())
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getInformant() == null || obs.getInformant().isEmpty())
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getParticipant() == null || obs.getParticipant().isEmpty())
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getReplacementOf() == null || obs.getReplacementOf().isEmpty())
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getReason() == null || obs.getReason().isEmpty())
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getReference() == null || obs.getReference().isEmpty())
+            .filter((RCMRMT030101UKObservationStatement obs) -> obs.getSequelTo() == null || obs.getSequelTo().isEmpty())
```

Comment:
I wonder if we could do anything to make this code simpler? It looks a bit repetitive. Not a major issue.